README.md

```diff
@@ -1 +1,67 @@
-# Weather-App
\ No newline at end of file
+# Weather-App
+Weather App to receive the weather data from any city in the world.
+
+## Screenshots
+
+![image](https://user-images.githubusercontent.com/33432289/93241617-390d4400-f74b-11ea-93f0-fb7696d52a9e.png)
+
+![image](https://user-images.githubusercontent.com/33432289/93241693-57733f80-f74b-11ea-990e-07ddcd18b0a4.png)
+
+## Built With
+
+- HTML5
+
+- Sass / CSS
+
+- Javascript
+
+- Node.js
+
+- Webpack
+
+- openWeather
+
+## Live Demo
+
+[Live version](https://weather-app-roberto.netlify.app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+```
+    git@github.com:LoboArkano/Weather-App.git
+```
+
+### Prerequisites
+
+- Web browser like Firefox or Chrome
+
```

Comment:
## Required changes.
- [ ] Let's add instructions on how to clone, and set up the project locally (furnish up us with guides on how to install the project's dependencacy and running it locally). Our README needs to be more descriptive enough on this aspect ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))